MoltenVK/MoltenVK/GPUObjects/MVKRenderPass.mm

```diff
@@ -600,11 +557,13 @@ static uint32_t getNextViewMaskGroup(uint32_t viewMask, uint32_t* startView, uin
 #if MVK_APPLE_SILICON
 	isMemorylessAttachment = attachment->getMTLTexture().storageMode == MTLStorageModeMemoryless;
 #endif
+	bool isResuming = mvkIsAnyFlagEnabled(_renderPass->getRenderingFlags(), VK_RENDERING_RESUMING_BIT);
 
 	// Only allow clearing of entire attachment if we're actually
 	// rendering to the entire attachment AND we're in the first subpass.
+	// If the renderpass was suspended, and is now being resumed, load the contents.
 	MTLLoadAction mtlLA;
-	if (loadOverride || !isRenderingEntireAttachment || !isFirstUseOfAttachment(subpass)) {
+	if (loadOverride || isResuming || !isRenderingEntireAttachment || !isFirstUseOfAttachment(subpass)) {
 		mtlLA = MTLLoadActionLoad;
```

Comment:
Yeah...it basically behaves like the other forced load/store overrides in these logic flows. 

Sadly ironic...because the point of the Vulkan suspend/resume renderpass flags is to do exactly the opposite...and NOT perform load/store activities. But I couldn't think of any other way of doing this within the architecture of Metal.

If you have any suggestions, let me know.